src/test/compile-fail/use-super-global-path.rs

```diff
@@ -14,10 +14,12 @@ struct S;
 struct Z;
 
 mod foo {
-    use ::super::{S, Z}; //~ ERROR global paths cannot start with `super`
+    use ::super::{S, Z}; //~ ERROR unresolved import `super`
+                         //~^ NOTE global paths cannot start with `super`
 
     pub fn g() {
-        use ::super::main; //~ ERROR global paths cannot start with `super`
+        use ::super::main; //~ ERROR unresolved import `super`
+                           //~^ NOTE global paths cannot start with `super`
```

Comment:
The notes are definitely an improvement -- but "unresolved import `super`" still feels misleading to me. Among other things, `super` is a keyword and hence could not *possibly* be an import, right? Can we readily change from adding a note to altering the main message?

Reply:
Done.